addons/resource.language.en_gb/resources/strings.po

```diff
@@ -13683,7 +13683,10 @@ msgctxt "#24047"
 msgid "This add-on can't be uninstalled"
 msgstr ""
 
-#empty string with id 24048
+#: xbmc/addons/GUIDialogAddonInfo.cpp
+msgctxt "#24048"
+msgid "%s is a system add-on"
```

Comment:
maybe bit more verbose like "%s is a system add-on and can't be removed"  or something.
Header already has  "This add-on can't be uninstalled" correct